datasafe-storage/datasafe-storage-impl-fs/src/test/java/de/adorsys/datasafe/business/impl/storage/FileSystemStorageServiceTest.java

```diff
@@ -37,6 +46,75 @@ void prepare(@TempDir Path dir) {
         );
     }
 
+    @Test
+    void objectExists() {
+        createFileWithMessage();
+        assertThat(storageService.objectExists(storageService.list(root).findFirst().get()));
+    }
+
+    @Test
+    void listEmpty() {
+        Path nonExistingFile = storageDir.resolve(UUID.randomUUID().toString());
+        AbsoluteLocation<PrivateResource> nonExistingFileLocation = new AbsoluteLocation<>(BasePrivateResource.forPrivate(nonExistingFile.toUri()));
+        assertThat(storageService.list(nonExistingFileLocation).collect(Collectors.toList()).isEmpty());
```

Comment:
Should be:
assertThat(storageService.list(nonExistingFileLocation)).isEmpty());